nucleus/base/tasks.py

```diff
@@ -0,0 +1,81 @@
+import json
+from copy import deepcopy
+from logging import getLogger
+
+from django.apps import apps
+from django.conf import settings
+from django.contrib.auth.models import User
+
+from github import UnknownObjectException
+from spinach.contrib.spinachd.tasks import tasks
+
+from nucleus.base import github
+
+
+log = getLogger(__name__)
+
+
+@tasks.task(name='save_to_github', max_retries=3)
```

Comment:
What happens if all tries fail?

Reply:
At the moment nothing. I think we'll need to setup some alerting on failures, as well as maybe increasing the failure limit. Spinach automatically does an exponential backoff thing with jitter, but it still wouldn't be that long even with several more attempts.

For this initial run my plan is to deploy this and watch it for a while to be sure it's keeping up before switching the site to use the new repo.

Reply:
Sounds like a plan!